ci/build-gdk.ps1

```diff
@@ -1,16 +1,21 @@
+# Expects gdk_home, which is not the GDK location in the engine
 param(
-  [string] $target_platform = "Win64"
+  [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine",
+  [string] $target_platform = "Win64",
+  [string] $build_output_dir
 )
 
 pushd "$($gdk_home)"
-
     Start-Event "build-unreal-gdk-$($target_platform)" "build-gdk"
     pushd "SpatialGDK"
-        $gdk_build_proc = Start-Process -PassThru -NoNewWindow -FilePath "$($gdk_home)\UnrealEngine\Engine\Build\BatchFiles\RunUAT.bat" -ArgumentList @(`
+        $gdk_build_proc = Start-Process -PassThru -NoNewWindow -FilePath "$unreal_path\Engine\Build\BatchFiles\RunUAT.bat" -ArgumentList @(`
             "BuildPlugin", `
-            " -Plugin=`"$($gdk_home)/SpatialGDK/SpatialGDK.uplugin`"", `
+            "-Plugin=`"$($gdk_home)/SpatialGDK/SpatialGDK.uplugin`"", `
             "-TargetPlatforms=$($target_platform)", `
-            "-Package=`"$($gdk_home)/SpatialGDK/Intermediate/BuildPackage/Win64`"" `
+                # Since we symlink the gdk into the engine, we have to put the build output outside of the GDK, 
```

Comment:
I think I'm missing some context on this. Why are we creating a GDK symlink into Engine?

Reply:
Reading through the rest of the PR makes this clearer :) I'm guessing it's done because otherwise we can't run Spatial tests? It would be really good to add this to wiki CI docs.

Reply:
The GDK gets symlinked into the engine Plugins folder because Setup.bat assumes that is where the GDK is placed. We want to use Setup.bat to avoid having duplicate functionality between it and the powershell script that CI was using.